src/psyclone/domain/lfric/transformations/raise_psyir_2_lfric_kern_trans.py

```diff
@@ -81,9 +81,11 @@ class RaisePSyIR2LFRicKernTrans(Transformation):
     nodes and symbols. This is currently limited to the specialisation
     of kernel metadata.
 
+    >>> from psyclone.configuration import Config
     >>> from psyclone.domain.lfric.transformations import \
             RaisePSyIR2LFRicKernTrans
     >>> from psyclone.psyir.frontend.fortran import FortranReader
+    >>> config = Config.get().api_conf("dynamo0.3")
```

Comment:
Otherwise doing:
`python -m doctest raise_psyir_2_lfric_kern_trans.py`
fails with:
`PSyclone internal error: LFRicConstants is being created before the config file is loaded`

Switching to master the doctest also fails with the same error, so this fix is not for an issue introduced by this PR.